parm/config/config.base.emc.dyn

```diff
@@ -140,7 +140,7 @@ export CASE="@CASECTL@"
 export CASE_ENKF="@CASEENS@"
 
 # Run with CCPP physics
-export RUN_CCPP="NO"
+export RUN_CCPP="YES"
```

Comment:
This should remain as "NO" by default. Users can change it to "YES" for their needs. This variable will likely change in the future so please change this back to "NO" for now. Thanks!

Reply:
After side discussions I am ok going with RUN_CCPP=YES now. Ignore this specific change request. Thanks!